src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -291,15 +304,21 @@ function togglePlayYearsWithData() {
                     <div class="px-2 border-end d-flex align-items-center">
                         <FontAwesomeIcon icon="grip-lines-vertical" />
                     </div>
-                    <div data-cy="time-slider-current-year" class="time-slider-bar-cursor-year">
-                        {{ currentYear }}
-                    </div>
+                    <input
+                        v-model="displayedYear"
+                        data-cy="time-slider-bar-cursor-year"
+                        maxlength="4"
+                        class="time-slider-bar-cursor-year"
+                        type="text"
+                        @input="setYearToInputIfValid"
+                    />
```

Comment:
The input should only allow numbers and a valid year between the min an max defined. If an invalid year is provided the input box should be marked as invalid in red using the bootstrap validation (see https://getbootstrap.com/docs/5.0/forms/validation/ and some example in the code).
Pressing enter should validate the year and exit the input editing.
On the year input we should not allow to grab and move the cursor as it make almost impossible to select the year by click and select.

Reply:
Is there a specific reason why the year should only be evaluated after pressing enter? I personally prefer if it auto validates after the input  contains four characters